support-frontend/assets/pages/[countryGroupId]/guardianAdLiteLanding/components/accordianFAQ.tsx

```diff
@@ -0,0 +1,247 @@
+						advertising cookies or similar technologies.
+					</p>
+					<p css={rowSpacing}>
+						A Guardian Ad-Lite subscription does not entitle you to the
+						additional benefits on offer via our All-access digital and Digital
+						+ print subscriptions, which are stated{' '}
+						<a href="/contribute">here</a>.
+					</p>
+				</div>
+			),
+		},
+		{
+			title: 'Will my Guardian Ad-Lite subscription work across all devices?',
+			body: (
+				<div css={rowSpacing}>
+					You can read the Guardian website without personalised advertising
+					across all devices by logging into your Guardian account. Guardian
+					Ad-Lite applies to our website only, and not the Guardian Live App.
+				</div>
+			),
+		},
+		{
+			title: 'How do I cancel my Guardian Ad-Lite subscription?',
+			body: (
+				<div css={rowSpacing}>
+					To cancel, go to Manage my account, and for further information on
+					your Guardian Ad-Lite subscription, see{' '}
+					<a href={guardianAdLiteTermsLink}>here</a>.
+				</div>
+			),
+		},
+		{
+			title: 'How do I contact customer services?',
+			body: (
+				<div css={rowSpacing}>
+					For any queries, including subscription-related queries, please visit
+					our <a href={helpCentreUrl}>Help centre</a>, where you will also find
+					contact details for your region.
+				</div>
+			),
+		},
+	],
+	SupporterPlus: [
+		{
+			title: 'Who is eligible for this discount?',
+			body: (
+				<div css={rowSpacing}>
+					Current students at [university/college] who register and verify
+					through Student Beans, are eligible for this discount.
+				</div>
+			),
+		},
+		{
+			title: 'What is included in my All-access subscription?',
+			body: (
+				<div css={rowSpacing}>
+					Your All-access digital subscription entitles you to all the benefits
+					listed above, including: unlimited access to the Guardian news app and
+					Guardian Feast app, ad-free reading on all your devices, exclusive
+					newsletter for supporters and far fewer asks for support.
+				</div>
+			),
+		},
+		{
+			title: 'Will my All-access subscription work across all devices?',
+			body: (
+				<div css={rowSpacing}>
+					You can access your All-access digital subscription across all devices
+					by logging into your Guardian account.
+				</div>
+			),
+		},
+		{
+			title: 'How can I manage my All-access subscription?',
+			body: (
+				<div css={rowSpacing}>
+					{' '}
+					To manage your subscription, go to Manage my account, and for further
+					information on your All-access digital subscription, see our Terms &
+					Conditions <a href={supporterPlusTermsLink}>here</a>
+				</div>
+			),
+		},
+		{
+			title: 'How do I contact customer services?',
+			body: (
+				<div css={rowSpacing}>
+					For any queries, including subscription-related queries, visit our{' '}
+					<a href={helpCentreUrl}>Help centre</a>
+				</div>
+			),
+		},
+	],
+};
+
+export type AccordianFAQProps = {
+	product: ActiveProductKey;
```

Comment:
Presumed this was the best prop at the moment to decide faq content. 

Currently the router only allows UK users to buy `GuardianAdLite` and the StudentLandingPage is limited to `AU`. so router seems best to aply these limitations at the moment.

This might need to be extended in future to include universities groups however.